script/playbackstrategy/modifiers/html5.js

```diff
@@ -35,6 +36,16 @@ define(
       var sentinelIntervalNumber;
       var lastSentinelTime;
 
+      var fakeTimer = {
+        currentTime: undefined,
+        runningTime: undefined,
+        window: {
```

Comment:
Would seekableRange be a more appropriate property name? It would be more consistent with the other usages

Reply:
Pulled out and moved into `restartable.js`.